### ISSUE TITLE
Numba "Failed at nopython" Error

### ISSUE THREAD

**Issue by andrewpaulreeves** [andrewpaulreeves/pyAOS#3]
@matthewtownson noticed a bug in the ``numba`` branch, where aoSimLib.zoom_numba would give 

```python
AttributeError: Failed at nopython (nopython mode backend)
'Module' object has no attribute 'global_variables'
```

This is confirmed on MacOSX, macports numba version 0.17.0. 

The error indicates that the function is not fully reducable down to native code, and python objects have been used as a fall back. As ``nopython`` is set, this calls an exception.

I'm sure it worked a couple of days ago, and it still works on linux builds. I did notice an update to llvm3.5 this morning, so perhaps its something to do with that....

**Comment by andrewpaulreeves** [comment]
Just installed the latest version of numba, which seems to have fixed this. Can you confirm @matthewtownson?

Will close for now

**Comment by matthewtownson** [comment]
I am still having issues, 0.17.0_1 (latest macports), which I don't think has been updated that recently. Are you just running the latest version of the master branch?

**Comment by andrewpaulreeves** [comment]
I cloned the numba branch, and sudo python setup.py installed that, after uninstalling the macports version. Hopefully when they package the soon to be released 1.8 for macports, that will fix it. I Think that it proves its not a PyAOS issue though, so don't really want this open. If this branch ever gets more official and its still a bug I can put a note in the README.
